src/configuration/sales/delivery-methods.md

```diff
@@ -66,6 +66,20 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sale
 |Show Method if Not Applicable|Website|Determines if Table Rates appears as an  option during checkout if the method does not apply to the purchase. Options: Yes / No|
 |Sort Order|Website|A number that determines the order that Table Rates appears when listed with other delivery methods during checkout.|
 
+### In-Store Delivery
+
+![]({% link images/images/in-store-delivery.png %}){: .zoom}
+[_In-Store Delivery_]({% link shipping/shipping-in-store-delivery.md %})
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Enabled|Website|When enabled, In-Store Delivery appears as an option in the Estimate Shipping and Tax section of the shopping cart, and in the Shipping section during checkout. Options: Yes / No|
+|Method Name|Store View|A name that describes the method of calculation that is used to produce a shipping estimate. The method name appears next to the calculated estimated rate in the shopping cart. The default value is "In-store Delivery”.|
+|Title|Store View|The name that is used for this shipping method during checkout.|
+|Price|Website|The price you charge the customer for a In-Store Delivery shipping.|
```

Comment:
```suggestion
|Price|Website|The price you charge the customer for in-store delivery shipping.|
```

Reply:
makes sense the simplification by Kevin, thanks to you 2!